v0/destinations/am/transform.js

```diff
@@ -445,7 +447,16 @@ function processSingleMessage(message, destination) {
           groupInfo.group_properties = groupTraits;
         } else {
           logger.debug("Group call parameters are not valid");
-          throw new CustomError("Group call parameters are not valid", 400);
+          throw new ErrorBuilder()
+            .setStatus(400)
+            .setMessage("Group call parameters are not valid")
+            .isExplicit(true)
+            .statsIncrement("transformation_and_proxy_errors", 1, {
+              destination: DESTINATION,
```

Comment:
Can we may be name this a little differently ? Like `rudderCategory` may be which indicates what does this stat actually belong to like does this stat belong to a source or to a destination.

Reply:
Ok if I am understanding correctly you want the metric name `transformation_and_proxy_errors` to be segregated as
`destinationTransform`,`sourceTransform`,and `proxy`?

Reply:
Sorry for misunderstanding, I get it now. You can continue with the same approach.